server/plugin/api.go

```diff
@@ -1289,23 +1296,26 @@ func (p *Plugin) createIssue(w http.ResponseWriter, r *http.Request, userID stri
 		return
 	}
 
-	rootID := issue.PostID
-	if post.RootId != "" {
-		rootID = post.RootId
+	postID := ""
+	channelID := issue.ChannelID
+	message := fmt.Sprintf("Created GitHub issue [#%v](%v)", result.GetNumber(), result.GetHTMLURL())
+	if post != nil {
+		postID = post.Id
+		channelID = post.ChannelId
+		message += fmt.Sprintf(" from a [message](%s)", permalink)
 	}
 
-	message := fmt.Sprintf("Created GitHub issue [#%v](%v) from a [message](%s)", result.GetNumber(), result.GetHTMLURL(), permalink)
 	reply := &model.Post{
```

Comment:
Does it make sense to have this message be an ephemeral message, specially when it is not attached to a post?
CC @aaronrothschild

Reply:
My reasoning was that it might be useful for other members in the channel to know that an issue had been created.

Reply:
I agree with @larkox here. Posting the message in the channel seems unexpected to me. I would like to go with the conservative route and use an ephemeral post.